src/components/menus/content-contextual-menu.js

```diff
@@ -267,6 +268,20 @@ const ContentContextualMenu = (props) => {
                                         handleDuplicateError(error.message);
                                     });
                                 break;
+                            case ElementType.SECURITY_ANALYSIS_PARAMETERS:
```

Comment:
you can avoid this duplication by re-using the call for voltage_init_parameters, because the only thing that change between the two request is activeElement.type
```
case ElementType.VOLTAGE_INIT_PARAMETERS:
case ElementType.SECURITY_ANALYSIS_PARAMETERS:
     duplicateParameter(
                                    newItemName,
                                    activeElement.type,
```